src/app/components/MessageBanner/index.tsx

```diff
@@ -12,12 +12,12 @@ import styles from './index.styles';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 interface MessageBannerProps {
-  summaries: Summary[];
+  summary: Summary;
   title: string;
   position: number;
 }
 
-const MessageBanner = ({ summaries, title, position }: MessageBannerProps) => {
+const MessageBanner = ({ summary, title, position }: MessageBannerProps) => {
```

Comment:
Is this also an opportunity to rename `title` to something like `heading` to prevent confusion with the summary title?
```suggestion
const MessageBanner = ({ summary, heading, position }: MessageBannerProps) => {
```